### DISCUSSION TITLE
Question about QuantityCodes

### DISCUSSION THREAD

**Discussion by djbox3** [stephanstapel/ZUGFeRD-csharp#520] [General]
Does the quantity code support other quantity codes in Rec20r13 and Rec20r16?

 QuantityCodes.C62 

Each

Fix

Gebühr

Honorar

hour

Minimum

Minutes

N/A

Pauschal

Stunde

Tage

word

Wörter

**Comment by stephanstapel** [comment]
You can find all available quantity codes along with decent documentation here:
https://github.com/stephanstapel/ZUGFeRD-csharp/blob/master/ZUGFeRD/QuantityCodes.cs

**Comment by tobitege** [comment]
Hallo @djbox3 
würde dies helfen (via KI)?

Hier sind die empfohlenen PEPPOL Quantity Codes für die genannten Begriffe:

- Each (Stück): EA (piece, unit)
- Fix (Festpreis): LS (lump sum)
- Gebühr: LS (lump sum)
- Honorar: LS (lump sum)
- Hour/Stunde: HUR (hour)
- Minimum: LS (lump sum)
- Minutes: MIN (minute)
- N/A: NIL oder "-" (wenn keine Einheit anwendbar)
- Pauschal: LS (lump sum)
- Tage: DAY (day)
- Word/Wörter: WRD (word)

Wichtige Anmerkungen:
- "Fix", "Gebühr", "Honorar" und "Pauschal" werden typischerweise als Pauschale (LS = lump sum) behandelt
- Für Zeiteinheiten sind die Codes eindeutig: HUR (Stunden), MIN (Minuten), DAY (Tage)
- Für Wörter gibt es den spezifischen Code WRD
- Bei Stückzahlen wird EA (each) verwendet
